bin/mlton-script

```diff
@@ -65,67 +49,100 @@ doitMLKit () {
     fi
 }
 
-doit () {
-    doitMLton "$@"
-    doitSMLNJ "$@"
-    doitPolyML "$@"
-    doitMLKit "$@"
-    echo 'Unable to run MLton.  Check that lib is set properly.' >&2
-    exit 1
-}
+
+# insert @MLton runtime arguments
```

Comment:
Maybe shift this `# insert @MLton runtime arguments` stuff to just before the `# insert compile arguments` to keep all of the `$@` adjustments together.